src/main/java/data_access/InMemoryUserDAO.java

```diff
@@ -104,4 +110,9 @@ public void loadUserFromFile(String userFilename) {
                     + "both src/main/resources/" + userFilename + " exist.");
         }
     }
+
+    @Override
+    public User getCurrentUser() {
+        return currentUser;
```

Comment:
could add a null-check for currentUser to handle case where no user is logged in

Reply:
Nice, I'll do that, thanks!

Reply:
Since all calling methods are already checking for null, added the `@Nullable` tag for any future code that might use this.